src/components/pages/Registration/RegisterFormComponents/StudentForm.js

```diff
@@ -41,6 +44,25 @@ export default function StudentForm({ studentArr, setStudentArr, idx }) {
     selectedAvatar === src ? 'selected-avatar' : '';
   // if the selected avatar matches the src of the clicked avatar, the class name will be set to selected avatar
 
+  const fileSelectedHandler = event => {
+    setSelectedFile(event.target.files[0]);
+  };
+
+  useEffect(() => {
+    const fd = new FormData();
+    if (selectedFile) {
+      fd.append('image', selectedFile, selectedFile.name);
+    }
+    // Code below will be used with endpoint for storing images
+    // axios.post('endpointForFileUploads', fd).then(res => {
+    // setFormValues({
+    //   ...formValues,
+    //   [avatarURL]: res,
+    // });
+    // updateStudent(formValues);
```

Comment:
Do we need this commented out code here?

Reply:
Well, as soon as we have the endpoint for the images it will be needed, so I just included it for anyone that may be working on the file after me. That being said, It can definitely just be removed and placed in once someone has the backend in place for it.